GUI/test/javafxgui/controller/GUIAppControllerTest.java

```diff
@@ -20,62 +20,70 @@
 import static jttt.core.game.GameType.*;
 import static org.junit.Assert.assertEquals;
 
-public class TTTControllerTest {
+public class GUIAppControllerTest {
 
     private final int DEFAULT_BOARD_DIMENSION = 3;
     private final int GUI_WINDOW_HEIGHT = 700;
     private final int GUI_WINDOW_WIDTH = 600;
     private GUIViewSpy gameViewSpy;
-    private TTTController controller;
+    private GUIAppController controller;
     private GUIGameMaker gameMaker;
+    private GUIGameMakerFake fakeGameMaker;
 
     @Before
     public void setUp() {
         gameViewSpy = new GUIViewSpy(new Scene(new StackPane(), GUI_WINDOW_HEIGHT, GUI_WINDOW_WIDTH));
-        gameMaker = new GUIGameMaker(new GUIPlayerFactory());
-        controller = new TTTController(gameViewSpy, gameMaker);
+        gameMaker = new GUIGameMaker();
+        fakeGameMaker = new GUIGameMakerFake();
+        controller = new GUIAppController(gameMaker, gameViewSpy);
         new JFXPanel();
     }
 
     @Test
     public void startGameWithGameTypeAndDimension() {
-        Controller controller = new TTTController(gameViewSpy, new GUIGameMaker(new GUIPlayerFactory()));
-        controller.startGame(GUI_HVC, DEFAULT_BOARD_DIMENSION);
+        GUIAppController controller = new GUIAppController(new GUIGameMaker(), gameViewSpy);
```

Comment:
duplication between this line and setUp, should you just use gameMaker? Do you need both a real GUIGameMaker and a fake?